sdks/python/setup.py

```diff
@@ -106,7 +106,7 @@ def get_version():
   'proto-google-cloud-datastore-v1==0.90.0',
   'googledatastore==7.0.0',
   # GCP packages required by tests
-  'google-cloud-bigquery>=0.22.1,<0.23',
+  'google-cloud-bigquery>=0.23.0,<1.0.0',
```

Comment:
Please put the upper bound to `0.24.0`. With major version `0` it is possible to introduce breaking changes with minor version changes.